.gitignore

```diff
@@ -1,3 +1,5 @@
 # created by virtualenv automatically
 venv
 .idea
+source
+destination
```

Comment:
I think these directories should stay - they are generic test files for basic script verification.